### ISSUE TITLE
"from skimage import io" crashing Anaconda with Python 3.6.6?

### ISSUE THREAD

**Issue by jefferton-sg** [scikit-image/scikit-image#3537]
Hi all! I'm noticing a strange issue on Windows 10, Ananconda python distribution (python version 3.6.6), scikit-image version 0.14.0. If I issue the command: 

`from skimage import io`

This makes Python crash. :-/ However, issuing:

`import skimage`

Works just fine. 

Anybody else having a similar issue? Or is there a workaround, where I can access the io package without specifically retrieving it with the from-import syntax? Oddly, this problem does not exist when I run the same command on my mac (but my mac is a laptop and I'm hoping to use a Windows 10 powerhouse for processing some large files)

**Comment by jni** [comment]
@jefferton-sg can you provide a traceback of the Python crash?

As to your second question, I suggest you try directly using [imageio](https://imageio.readthedocs.io/en/stable/)

**Comment by jefferton-sg** [comment]
Thanks for the workaround tip! I tried importing imageio directly and it works perfectly. 

I must apologize but I am still very new to Python... how can I log the traceback? I tried wrapping the import statement in a try/except but it doesn’t trigger the exception... calling the import just causes python to quietly exit.

**Comment by jni** [comment]
Oh! Wow! I have never seen that. No segfault, abort trap, anything?

**Comment by hmaarrfk** [comment]
Are you in spyder or jupyter? Can you try running the python interpreter from the anaconda prompt?

**Comment by jefferton-sg** [comment]
![img_0326](https://user-images.githubusercontent.com/37162010/48047886-64d24280-e14e-11e8-94fc-e3234066a111.PNG)
Yeah it's really strange... I tried issuing:

`from skimage import io`

from the Anaconda prompt directly and it just exits. (see screenshot)

**Comment by stefanv** [comment]
Can you import matplotlib successfully?  `import matplotlib.pyplot as plt; plt.plot([1,2,3])` ?

**Comment by hmaarrfk** [comment]
best screenshot ever, thanks. I have no words.

**Comment by soupault** [comment]
Wait. (stupid question section) Isn't that 32-bit OS running Python compiled for 64-bit? 🤔

**Comment by jefferton-sg** [comment]
@stefanv That is a very good point - this statement produces the same behavior. Opening the Anaconda prompt and issuing 

`import matplotlib.pyplot as plt`

also causes Python to quietly exit. (see screenshot... another "amazing" one? ha!)

<img width="675" alt="capture" src="https://user-images.githubusercontent.com/37162010/48080962-d1c3f780-e1a3-11e8-9002-15570723fa8e.PNG">

@soupault You also make a good point, however this should be a 64-bit version of Windows. Checking my system configuration confirms this, also attaching a screenshot. Though Python quoting win32 is indeed a little suspicious...?
<img width="356" alt="capture2" src="https://user-images.githubusercontent.com/37162010/48080963-d1c3f780-e1a3-11e8-9c7a-f8f3753f15e0.PNG">

**Comment by soupault** [comment]
@jefferton-sg could you try to repeat the experiment in `python -vvv`, not just plain `python`?

**Comment by jefferton-sg** [comment]
@soupault Sure! Here is the output pasted into a text file. I separated it into three files: one just with the output of launching `python -vvv`, the other with the output of issuing `from skimage import io`, and a third with the output of issuing `import matplotlib.pyplot as plt`.
[importing_io_with_vvv.txt](https://github.com/scikit-image/scikit-image/files/2559781/importing_io_with_vvv.txt)
[launching-python_with_vvv.txt](https://github.com/scikit-image/scikit-image/files/2559782/launching-python_with_vvv.txt)
[importing_matplotlib_with_vvv.txt](https://github.com/scikit-image/scikit-image/files/2559783/importing_matplotlib_with_vvv.txt)

**Comment by soupault** [comment]
Thanks! So, it seems like PyQt5 may cause the issue:
```
# C:\Users\User\Anaconda3\lib\site-packages\PyQt5\__pycache__\__init__.cpython-36.pyc matches C:\Users\User\Anaconda3\lib\site-packages\PyQt5\__init__.py
# code object from 'C:\\Users\\User\\Anaconda3\\lib\\site-packages\\PyQt5\\__pycache__\\__init__.cpython-36.pyc'
import 'PyQt5' # <_frozen_importlib_external.SourceFileLoader object at 0x0000022931F7EDA0>
# trying C:\Users\User\Anaconda3\lib\site-packages\PyQt5\QtCore.cp36-win_amd64.pyd
# trying C:\Users\User\Anaconda3\lib\site-packages\PyQt5\QtCore.pyd

(base) C:\Users\User>
```
Could you:
1. Uninstall PyQt5 and try to import matplotlib and/or scikit-image;
2. Reinstall PyQt5 (and clear the corresponding `__pycache__` dir) and --//--.

**Comment by jefferton-sg** [comment]
@soupalt I uninstalled PyQt and that uninstalled scikit-image and matplotlib, along with a few other packages... so importing either skimage or matplotlib after removing pyqt reported a ModuleNotFoundError. BUT! Then I reinstalled pyqt, along with scikit-image, matplotlib, and all the other packages that were removed with pyqt (anaconda-navigator, jupyter, navigator-updater, qtconsole, seaborn, spyder, and tomopy), and now: success! I am happily importing io from skimage. Thanks so much for the help!! (I wonder what went awry over here in the first place...?)
